cmd/plan/pl005/plan.go

```diff
@@ -20,12 +20,10 @@ var Plan = []plan.Step{
 	{
 		Action:  "update/cluster/major",
 		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
-		Comment: "trigger major upgrade",
```

Comment:
These are the only comments left. I am wondering how useful they are now that the actions do somehow speak for themselves. I am open for suggestions. If someone wants to keep the comments let me know.